etc/adminhtml/system.xml

```diff
@@ -246,6 +246,10 @@
                     <label>Send shipping details</label>
                     <source_model>Magento\Config\Model\Config\Source\Yesno</source_model>
                 </field>
+                <field id="split_auto_cancel" type="select" sortOrder="31" translate="label" showInDefault="1" showInWebsite="1" showInStore="1">
+                    <label>Cancel split-pay declined orders</label>
```

Comment:
I think it might be helpful to add a little more context here with an explanation underneath to make sure the user knows exactly what split pay is without needing to reference any docs. If it seems too long to make a full explanation, then we can just have a small reminder note and say to reference the docs for more info

Reply:
Sure, just let me know what text should go in it